test/plugins/CMakeLists.txt

```diff
@@ -18,6 +18,8 @@ set (test_plugins
   Null
 )
 
+add_subdirectory(sensor_demo)
```

Comment:
Could you rename this to something that alludes to custom rendering sensors?

Reply:
Changed the name of directory to ``custom_rendering_sensor_test``